src/devtools/client/webconsole/components/FilterBar/FilterBar.js

```diff
@@ -37,16 +37,13 @@ class FilterBar extends Component {
   static get propTypes() {
     return {
       closeButtonVisible: PropTypes.bool,
-      closeSplitConsole: PropTypes.func,
-      dispatch: PropTypes.func.isRequired,
```

Comment:
Looks like we still need this for the child (<ConsoleSettings />).

Reply:
Now that I looked into it, the <FilterBar />'s parent doesn't appear to be passing in a dispatch prop. Am I missing something here?

https://github.com/RecordReplay/devtools/blob/cebb7ffa910351227c153f330b84b2fbf750c0f4/src/devtools/client/webconsole/components/App.js#L154-L158

Reply:
Good catch. I added dispatch